### DISCUSSION TITLE
Toggling Mute for Active Sources?

### DISCUSSION THREAD

**Discussion by iua2** [cpyarger/obs-midi#148] [Q&A] [answered]
Have just discovered obs-midi and it is very powerful!
I am trying to figure out if it will allow toggling mute between several active sources? I have for example 5 separate browser sources active (all muted by default) and want to rotate my midi controller knob and toggle mute for each active source. I only want **one** of the active sources unmuted at a time. If I make 2 of the 5 sources inactive, the knob would toggle mute between the remaining 3. If 8 sources were active it would select between the 8, etc.

Can this be done?

**Comment by cpyarger** [comment, marked as answer]
Not currently. but I can see this being a part of the uppcoming macros system

**Reply by iua2** [reply]
Thanks! How do I learn more about the macros system?

**Reply by cpyarger** [reply]
It is still being worked on. I will post documentation on how to use it once it is released
